nautobot/project-static/js/table_sorting_indicator.js

```diff
@@ -0,0 +1,51 @@
+document.addEventListener("DOMContentLoaded", function() {
+    // Select all header elements with the class "orderable" containing links
+    const headers = document.querySelectorAll(".orderable a");
+    const clickCounts = {};
+
+    headers.forEach(header => {
+        // Get the column name from the data attribute
+        const columnName = header.dataset.columnName;
+        clickCounts[columnName] = 0;
+
+        // Add a click event listener to each header
+        header.addEventListener("click", function(event) {
+            // Prevent the default link behavior
+            event.preventDefault();
+
+            // Increment the click count for the column
+            clickCounts[columnName] += 1;
```

Comment:
Unless I'm missing something, this variable isn't actually used for anything - left over from a previous implementation?

Reply:
good catch! thank you :D